CHANGELOG.md

```diff
@@ -9,7 +9,9 @@
 
 ##### Enhancements
 
-* None.
+* Add `whitelist_rules` rule whitelists in config files  
+  [Daniel Beard](https://github.com/daniel-beard)
+  [#436](https://github.com/realm/SwiftLint/issues/436)
```

Comment:
This should be the issue number for the issue this PR resolves, not the PR number. This is explained in CONTRIBUTING.md.